orchestrator/services/processes.py

```diff
@@ -108,7 +107,7 @@ def _db_create_process(stat: ProcessStat) -> None:
         workflow_id=wf_table.workflow_id,
         last_status=ProcessStatus.CREATED,
         created_by=stat.current_user,
-        is_task=stat.workflow.target == Target.SYSTEM,
+        is_task=stat.workflow.is_task,
```

Comment:
As discussed, better to change this to `wf_table.is_task` to use the value from the `WorkflowTable` DB model. 
And remove the default/computed `is_task` value from the `Workflow(Protocol)`.
So that there is exactly 1 definition of whether a workflow is a task